packages/adminrouter/extra/src/Makefile.common

```diff
@@ -109,21 +111,26 @@ api-docs: clean-devkit-container devkit
 	done
 
 .PHONY: check-api-docs
+DOCS_TEXT := See https://github.com/dcos/dcos/tree/master/packages/adminrouter/extra/src\#endpoints-documentation for more info.
 check-api-docs: api-docs
-	@git update-index --refresh || (echo 'Found local changes. Please run `make api-docs` and commit the result.' >&2 && exit 1)
-	@git diff-files --quiet || (echo 'Found unstaged changes. Please run `make api-docs` and commit the result.' >&2 && exit 1)
-	@git ls-files --exclude-standard --others || (echo 'Found untracked and unignored changes. Please run `make api-docs` and commit the result.' >&2 && exit 1)
+ifneq ($(shell git update-index --refresh; echo $$?),0)
+	$(error "Found local changes. Please run `make api-docs` and commit the result. $(DOCS_TEXT)")
+endif
+ifneq ($(shell git diff-files --quiet; echo $$?),0)
+	$(error 'Found unstaged changes. Please run `make api-docs` and commit the result. $(DOCS_TEXT)')
+endif
+ifneq ($(shell git ls-files --exclude-standard --others; echo $$?),0)
+	$(error 'Found untracked and unignored changes. Please run `make api-docs` and commit the result. $(DOCS_TEXT)')
+endif
 	@echo 'No changes found -- Admin Router API docs are up to date.' >&2
 
-.PHONY: flake8
-flake8: clean-devkit-container devkit
-	#FIXME - split it into two targets
-	docker run \
+.PHONY: lint
+lint: clean-devkit-container devkit
+	@#FIXME - split it into two targets
+	@docker run \
 		$(DEVKIT_DOCKER_OPTS) \
-		mesosphere/adminrouter-devkit:full /bin/bash -x -c " \
-			flake8 -v \
- 		"
+		mesosphere/adminrouter-devkit:full /bin/bash -x -c "flake8"
```

Comment:
Nothing changed here when it comes to an/the error message, it is still printed. 

Output when:
* syntax is OK,  `-v` is specified on the commandline:
```
flake8.plugins.manager    MainProcess    129 INFO     Loading entry-points for "flake8.extension".
flake8.plugins.manager    MainProcess    168 INFO     Loading entry-points for "flake8.listen".
flake8.plugins.manager    MainProcess    168 INFO     Loading entry-points for "flake8.report".
flake8.plugins.manager    MainProcess    168 INFO     Loading plugin "C90" from entry-point.
flake8.plugins.manager    MainProcess    169 INFO     Loading plugin "pycodestyle.python_3000_not_equal" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.imports_on_separate_lines" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.python_3000_raise_comma" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.whitespace_around_operator" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.tabs_or_spaces" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.continued_indentation" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.comparison_type" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.whitespace_around_keywords" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.missing_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.missing_whitespace_after_import_keyword" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "pycodestyle.comparison_negative" from entry-point.
flake8.plugins.manager    MainProcess    174 INFO     Loading plugin "F" from entry-point.
flake8.plugins.manager    MainProcess    185 INFO     Loading plugin "pycodestyle.whitespace_around_named_parameter_equals" from entry-point.
flake8.plugins.manager    MainProcess    185 INFO     Loading plugin "pycodestyle.whitespace_before_parameters" from entry-point.
flake8.plugins.manager    MainProcess    185 INFO     Loading plugin "pycodestyle.maximum_line_length" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.explicit_line_join" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.comparison_to_singleton" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.missing_whitespace_around_operator" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.compound_statements" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.trailing_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.python_3000_backticks" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.extraneous_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.break_around_binary_operator" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.tabs_obsolete" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.whitespace_before_comment" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.indentation" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.module_imports_on_top_of_file" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.python_3000_has_key" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.whitespace_around_comma" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.trailing_blank_lines" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "pycodestyle.blank_lines" from entry-point.
flake8.plugins.manager    MainProcess    186 INFO     Loading plugin "I00" from entry-point.
flake8.plugins.manager    MainProcess    192 INFO     Loading plugin "quiet-nothing" from entry-point.
flake8.plugins.manager    MainProcess    193 INFO     Loading plugin "default" from entry-point.
flake8.plugins.manager    MainProcess    193 INFO     Loading plugin "quiet-filename" from entry-point.
flake8.plugins.manager    MainProcess    193 INFO     Loading plugin "pylint" from entry-point.
flake8.checker            MainProcess    205 INFO     Making checkers
flake8.checker            MainProcess    247 INFO     Checking 39 files
flake8.processor          ForkPoolWorker-3    250 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    250 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    260 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    260 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-2    262 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-3    264 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    278 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    282 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    291 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    292 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-8    293 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    319 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    329 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-4    337 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    344 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    347 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-8    350 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    350 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    357 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-3    359 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    361 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-2    364 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    383 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    397 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    399 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    405 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    430 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    437 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-3    445 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    459 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-2    461 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    476 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    493 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-8    509 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-4    527 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-2    539 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    544 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-8    580 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    624 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.main.application   MainProcess    718 INFO     Finished running
flake8.main.application   MainProcess    719 INFO     Reporting errors
flake8.main.application   MainProcess    721 INFO     Found a total of 90 violations and reported 0
```
* syntax is OK,  `-v` is NOT specified on the commandline:
```
root@7f90cd33b009:/usr/local/adminrouter/nginx/conf# flake8
root@7f90cd33b009:/usr/local/adminrouter/nginx/conf#
```
* syntax is not formatted properly, -v is specified on the commandline:
```
flake8.plugins.manager    MainProcess    143 INFO     Loading entry-points for "flake8.extension".
flake8.plugins.manager    MainProcess    187 INFO     Loading entry-points for "flake8.listen".
flake8.plugins.manager    MainProcess    188 INFO     Loading entry-points for "flake8.report".
flake8.plugins.manager    MainProcess    188 INFO     Loading plugin "C90" from entry-point.
flake8.plugins.manager    MainProcess    188 INFO     Loading plugin "F" from entry-point.
flake8.plugins.manager    MainProcess    200 INFO     Loading plugin "pycodestyle.missing_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.whitespace_around_operator" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.imports_on_separate_lines" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.blank_lines" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.whitespace_around_comma" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.whitespace_before_parameters" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.python_3000_has_key" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.break_around_binary_operator" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.comparison_negative" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.tabs_or_spaces" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.python_3000_backticks" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.trailing_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.maximum_line_length" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.comparison_type" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.python_3000_raise_comma" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.explicit_line_join" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.compound_statements" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.missing_whitespace_around_operator" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.whitespace_before_comment" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.comparison_to_singleton" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.trailing_blank_lines" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.python_3000_not_equal" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.module_imports_on_top_of_file" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.extraneous_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    205 INFO     Loading plugin "pycodestyle.indentation" from entry-point.
flake8.plugins.manager    MainProcess    206 INFO     Loading plugin "pycodestyle.missing_whitespace_after_import_keyword" from entry-point.
flake8.plugins.manager    MainProcess    206 INFO     Loading plugin "pycodestyle.whitespace_around_named_parameter_equals" from entry-point.
flake8.plugins.manager    MainProcess    206 INFO     Loading plugin "pycodestyle.whitespace_around_keywords" from entry-point.
flake8.plugins.manager    MainProcess    206 INFO     Loading plugin "pycodestyle.tabs_obsolete" from entry-point.
flake8.plugins.manager    MainProcess    206 INFO     Loading plugin "pycodestyle.continued_indentation" from entry-point.
flake8.plugins.manager    MainProcess    206 INFO     Loading plugin "I00" from entry-point.
flake8.plugins.manager    MainProcess    212 INFO     Loading plugin "pylint" from entry-point.
flake8.plugins.manager    MainProcess    212 INFO     Loading plugin "default" from entry-point.
flake8.plugins.manager    MainProcess    212 INFO     Loading plugin "quiet-nothing" from entry-point.
flake8.plugins.manager    MainProcess    212 INFO     Loading plugin "quiet-filename" from entry-point.
flake8.checker            MainProcess    224 INFO     Making checkers
flake8.checker            MainProcess    266 INFO     Checking 39 files
flake8.processor          ForkPoolWorker-3    269 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    270 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    282 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    283 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-2    284 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-3    286 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    303 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    308 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    318 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-8    326 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    355 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    365 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-4    366 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-3    368 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-8    382 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    386 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    387 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-2    391 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    392 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    395 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    397 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    410 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    418 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    421 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    427 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    453 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-3    455 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    458 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-5    481 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    513 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-2    520 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    521 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-6    536 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-2    538 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-4    552 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-1    580 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-7    594 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-8    660 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.processor          ForkPoolWorker-8    793 WARNING  Plugin requested optional parameter "search_current" but this is not an available parameter.
flake8.main.application   MainProcess    833 INFO     Finished running
flake8.main.application   MainProcess    834 INFO     Reporting errors
flake8.main.application   MainProcess    836 INFO     Found a total of 91 violations and reported 1
./test-harness/tests/test_agent.py:7:96: E501 line too long (107 > 95 characters)
```
* syntax is not formatted properly, -v is NOT specified on the commandline:
```
root@7f90cd33b009:/usr/local/adminrouter/nginx/conf# flake8
./test-harness/tests/test_agent.py:7:96: E501 line too long (107 > 95 characters)
```